src/ci/bin/test.inc.sh

```diff
@@ -816,6 +816,14 @@ cromwell::private::install_wait_for_it() {
     chmod +x "$CROMWELL_BUILD_WAIT_FOR_IT_SCRIPT"
 }
 
+cromwell::private::install_vault() {
+    curl \
+        --location --fail --silent --show-error \
+        --output "${CROMWELL_BUILD_VAULT_ZIP}" \
+        "https://releases.hashicorp.com/vault/1.6.3/vault_1.6.3_${CROMWELL_BUILD_OS}_amd64.zip"
```

Comment:
Should we perhaps stage this into one of our own buckets? It feels like this might be hitting that download link pretty hard...

Reply:
🤷 Happy to use another source.

We hit docker.io pretty hard even after logging in, downloading gigs of data. This download of `vault` is ~30 downloads per PR, and the download link is from the instructions on hashicorp's webpage. So I assume they have `releases.hashicorp.com` setup with a CDN.

- https://learn.hashicorp.com/tutorials/vault/getting-started-install
- https://www.vaultproject.io/docs/install